content/en/serverless/guide/serverless_warnings.md

```diff
@@ -126,6 +126,12 @@ At least one invocation in the selected time range approached the maximum durati
 
 **Resolution:** Lambda functions approaching the maximum timeout limit of 15 minutes risk termination by the Lambda runtime. This could lead to slow or failed responses to incoming requests. Consider improving the performance of your Lambda function, using smaller lambdas in a Step Function, or moving your workload to a longer running environment like ECS Fargate.
 
+### Recursive Invocations Dropped
+
+Invocations in this function have a recursive loop, generally caused by recursive triggering between AWS entities (e.g., Lambda => SQS => Lambda). When this exceeds your maxReceiveCount (default 16), then it adds to this metric. More information can be found [here](15).
```

Comment:
```suggestion
Invocations in this function have a recursive loop, generally caused by recursive triggering between AWS entities (for example, Lambda -> SQS -> Lambda). When this exceeds your `maxReceiveCount` (default 16), then it adds to this metric. For more information, see [Use Lambda recursive loop detection to prevent infinite loops][15].
```